javascript/ql/src/semmle/javascript/frameworks/SQL.qll

```diff
@@ -31,11 +31,17 @@ private module MySql {
   /** Gets the package name `mysql` or `mysql2`. */
   API::Node mysql() { result = API::moduleImport(["mysql", "mysql2"]) }
 
+  /** Gets a reference to `mysql.createConnection`. */
+  API::Node createConnectionCallee() { result = mysql().getMember("createConnection") }
+
   /** Gets a call to `mysql.createConnection`. */
-  API::Node createConnection() { result = mysql().getMember("createConnection").getReturn() }
+  API::Node createConnection() { result = createConnectionCallee().getReturn() }
+
+  /** Gets a reference to `mysql.createPool`. */
+  API::Node createPoolCallee() { result = mysql().getMember("createPool") }
```

Comment:
Again, I'd suggest calling this predicate `createPool` and renaming the predicate below, perhaps to `pool()` (by analogy with `connection()`).